src/photon/photon_algorithm.c

```diff
@@ -156,24 +232,20 @@ void fetch_missing_dependencies(local_scheduler_state *state,
   for (int i = 0; i < num_args; ++i) {
     if (task_arg_type(spec, i) == ARG_BY_REF) {
       object_id obj_id = task_arg_id(spec, i);
-      available_object *entry;
-      HASH_FIND(handle, algorithm_state->local_objects, &obj_id, sizeof(obj_id),
+      object_entry *entry;
+      HASH_FIND(hh, algorithm_state->local_objects, &obj_id, sizeof(obj_id),
                 entry);
       if (entry == NULL) {
         /* The object is not present locally, fetch the object. */
-        object_id object_ids[1] = {obj_id};
-        plasma_fetch(state->plasma_conn, 1, object_ids);
-        /* Create a fetch request and add a timer to the event loop to ensure
-         * that the fetch actually happens. */
-        fetch_object_request *fetch_req = malloc(sizeof(fetch_object_request));
+        plasma_fetch(state->plasma_conn, 1, &obj_id);
+        /* Create an entry and add it to the list of active fetch requests to
+         * ensure that the fetch actually happens. */
+        object_entry *fetch_req = malloc(sizeof(object_entry));
         fetch_req->object_id = obj_id;
-        fetch_req->state = state;
-        fetch_req->algorithm_state = algorithm_state;
-        fetch_req->timer = event_loop_add_timer(
-            state->loop, LOCAL_SCHEDULER_FETCH_TIMEOUT_MILLISECONDS,
-            fetch_object_timeout_handler, fetch_req);
-        /* The fetch request will be freed and removed from the hash table in
-         * handle_object_available when the object becomes available locally. */
+        /* The fetch request will be moved to the hash table of locally
+         * available objects in
+         * handle_object_available when the object becomes available locally.
```

Comment:
can you change the line breaks in this comment so that the second line isn't so short?